scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java

```diff
@@ -20,4 +21,13 @@ void increment() {
         counter.incrementAndGet();
     }
     
-}
\ No newline at end of file
+    @Scheduled(cron="0 15 10 * * ?")
```

Comment:
Would it be possible to make it testable and extend the tests?

Reply:
I am thinking about incrementing the counter for each of the two added methods. Is that acceptable?

Reply:
+1

Reply:
Done, I have updated the two methods with a call that increment the counter. 
Same modification applied in the guide.